### PR TITLE
Add testing env and Tabs, PageHeader, EmptyState and Table tests

### LINKED ISSUE #860: Add unit tests for Table
Table has a bunch of logic and decisions sitting inside of it.

Our visual changes can't catch bugs in them, we'll have to write unit tests with jest + enzyme

### LINKED ISSUE #861: Add unit tests for Tabs
Tabs have some logic and decisions sitting inside of them.

Our visual changes can't catch bugs in them, we'll have to write unit tests with jest + enzyme

### PR DESCRIPTION
**Pull request by francocorreasosa**
Resolves #860 
Resolves #861

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@auth0/babel-preset-cosmos"]
```
Comment (siddharthkp) [inline .babelrc]: I don't think you should drop this into root, that can lead to all the other internal configs falling back to this

File: package-scripts.js [hunk 2]
```diff
@@ -46,8 +46,8 @@ module.exports = {
     },
     test: {
       default: {
-        script: parallel('production.build', 'test.ci'),
-        description: 'Check if applications build + Run visual tests'
+        script: parallel('production.build', 'test.ci', 'test.unit'),
+        description: 'Check if applications build + Run visual tests + Run unit tests'
```
Comment (siddharthkp) [inline package-scripts.js]: perfect 💯

File: test/package.json [hunk 3]
```diff
@@ -7,20 +7,24 @@
   "license": "MIT",
   "scripts": {
     "test": "jest --no-cache",
+    "test-watch": "jest --watch --no-cache",
```
Comment (siddharthkp) [inline test/package.json]: Can we please move `test` to `internal/test`?

File: test/unit/tabs.test.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+describe('Tabs tests', () => {
+  it('renders only selected tab content', () => {
+    const { generator, content } = tabsFactory()
+
+    // Create three <Tabs> instances with different tabs selected
+    const [first, second, third] = [generator(0), generator(1), generator(2)]
+
+    expect(first.contains(content.first)).toBe(true)
```
Comment (siddharthkp) [inline test/unit/tabs.test.js]: Do you think it would easier to maintain snapshot tests for rendering stuff?

File: test/unit/tabs.test.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+      <Tabs selected={0} onSelect={() => {}}>
+        <Tabs.Tab title="First title">First content</Tabs.Tab>
+        <Tabs.Tab title="Second Title">Second Content</Tabs.Tab>
+      </Tabs>
+    )
+    const tabLinks = wrapper.find(TabLink)
+    expect(tabLinks).toHaveLength(2)
+  })
+
+  it('onSelect is called on tab title click', () => {
```
Comment (siddharthkp) [inline test/unit/tabs.test.js]: This is the one case we should write functional/expect tests for.

File: test/unit/tabs.test.js (hunk 4 above)
Comment (francocorreasosa) [reply test/unit/tabs.test.js]: Yes! Changed it :)

**Pull request merged**